cups/translate-attrs_test.go

```diff
@@ -207,13 +207,58 @@ func TestConvertMarkers(t *testing.T) {
 	}
 
 	pt = map[string][]string{
-		attrMarkerNames:  []string{"black", "color", "rainbow", "zebra", "pony"},
-		attrMarkerTypes:  []string{"toner", "toner", "ink", "staples", "water"},
+		attrMarkerNames:  []string{"black", "color"},
+		attrMarkerTypes:  []string{"toner", "toner", "ink"},
+		attrMarkerLevels: []string{"10", "11", "12"},
+	}
+	m, ms = convertMarkers(pt)
+	if m != nil {
+		t.Logf("expected nil")
+		t.Fail()
+	}
+	if ms != nil {
+		t.Logf("expected nil")
+		t.Fail()
+	}
+
+	pt = map[string][]string{
+		attrMarkerNames:  []string{"black", "color", "rainbow"},
+		attrMarkerTypes:  []string{"toner", "toner"},
+		attrMarkerLevels: []string{"10", "11", "12"},
+	}
+	m, ms = convertMarkers(pt)
+	if m != nil {
+		t.Logf("expected nil")
+		t.Fail()
+	}
+	if ms != nil {
+		t.Logf("expected nil")
+		t.Fail()
+	}
+
+	pt = map[string][]string{
```

Comment:
I'm not sure how I managed to do that, but I duplicated one test and didn't notice before I submitted.

What was supposed to be done was add three tests:

1. one with attrMarkerNames a different length than attrMarkerLevels
2. one with attrMarkerTypes a different length than attrMarkerLevels (I accidentally duplicated this one)
3. one with extra vendor information in the name and type to test the functionality of the new function `fixMarkers`

So maybe once you finish reviewing the rest of the code, I can fix this issue, rebase, and send another PR? Not sure really what the best way to fix a PR is...